test/ws-container.install

```diff
@@ -16,6 +16,13 @@ system
 for rpm in ws bridge tests $PACKAGES; do
     rpm2cpio /var/tmp/cockpit-$rpm-*.rpm | cpio -i --make-directories --directory=/var/tmp/install
 done
+
+# HACK: Fedora CoreOS is already on Python 3.13, while Fedora-40 is Python3.12
+if [ -d /var/tmp/install/usr/lib/python3.13/site-packages/cockpit ]; then
+    mkdir -p /var/tmp/install/usr/lib/python3.12/site-packages/
+    mv /var/tmp/install/usr/lib/python3.13/site-packages/cockpit /var/tmp/install/usr/lib/python3.12/site-packages/
+fi
```

Comment:
Super ugly (also, you could just have renamed the python3.13 directory), and doesn't generalize, but maybe that's a good thing. The *clean* way to fix this would be to build rpms on f40 for the container, and on f41 for the fcos host. But since this is version specific, it'll bite us again in the future, i.e. we have a reminder to clean this up. :smirk: